docs/proposals/002-api-proposal/proposal.md

```diff
@@ -336,8 +318,8 @@ Our original idea was to define all LLMService config at the Kubernetes Gateway
 
 - Reasonable defaults (how do we behave in the absence of user-specified values in optional fields)
```

Comment:
The rest of the FAQs are pretty confusing or outdated (the SLO part) to me. Perhaps we can delete them, or add some more detail/example to explain them better?